packages/gnutls/SPECS/gnutls.spec

```diff
@@ -100,38 +159,71 @@ echo "SYSTEM=NORMAL" >> tests/system.prio
 # via the crypto policies
 
 %build
-export CCASFLAGS="$CCASFLAGS -Wa,--generate-missing-build-notes=yes"
-
-./configure --prefix=/usr/sgug \
```

Comment:
Here you used plain `./configure` rather than using `%configure` - using the rpm macro means it auto-passes things like the correct prefix and libdir etc. Try and stick to the original spec if you can, normally all that is done for you.